modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -110,7 +94,7 @@ export default {
       }
     }
   },
-  mounted() {
+  created() {
```

Comment:
I don't know why, but on "mounted" some fields were `undefined`. With `created` everything field has a value with `{ data: undefined, error: false }` so no problem anymore. If anyone has an explanation..

Reply:
Weird. Mounted is actually called later.